lib/spotlight/dor/indexer.rb

```diff
@@ -53,6 +54,21 @@ def resource druid
 
     private
 
+    # add the box number to solr_doc as box_ssi field (note: single valued!)
```

Comment:
Could we group the series/box/folder methods together? It's kinda funny to have them interspersed with unrelated logic.